difflint/lint.py

```diff
@@ -0,0 +1,62 @@
+import os.path
+from pkg_resources import resource_filename
+import shutil
+import subprocess
+
+
+def check_linters():
+    """Checks that all required external executables are in the PATH. Returns a
+    list of executables that are not, or an empty list if everything is OK."""
+    required = ['jscs', 'jshint']
```

Comment:
I didn't place it at the top of the file since it's not meant for use outside this function; just a notational simplification. However, I did capitalize it.